src/world/area_flo/flo_18/npc.c

```diff
@@ -422,45 +442,69 @@ EvtScript N(EVS_GenericBombReaction) = {
 };
 
 EvtScript N(EVS_BombReaction_Magikoopa) = {
+#if VERSION_JP
+    Set(MV_ReactingNpc, 0)
+    Call(GetNpcPos, NPC_Magikoopa, LVar0, LVar1, LVar2)
+    ExecWait(N(EVS_GenericBombReaction))
+#else
     IfEq(MV_ReactingNpc, -1)
         Set(MV_ReactingNpc, 0)
         Call(GetNpcPos, NPC_Magikoopa, LVar0, LVar1, LVar2)
         ExecWait(N(EVS_GenericBombReaction))
         Set(MV_ReactingNpc, -1)
     EndIf
+#endif
     Return
     End
 };
 
 EvtScript N(EVS_BombReaction_Lakitu_01) = {
+#if VERSION_JP
+    Set(MV_ReactingNpc, 1)
+    Call(GetNpcPos, NPC_Lakitu_01, LVar0, LVar1, LVar2)
+    ExecWait(N(EVS_GenericBombReaction))
+#else
     IfEq(MV_ReactingNpc, -1)
         Set(MV_ReactingNpc, 1)
         Call(GetNpcPos, NPC_Lakitu_01, LVar0, LVar1, LVar2)
         ExecWait(N(EVS_GenericBombReaction))
         Set(MV_ReactingNpc, -1)
     EndIf
+#endif
```

Comment:
Maybe better to do

```suggestion
#if !VERSION_JP
    IfEq(MV_ReactingNpc, -1)
#endif
        Set(MV_ReactingNpc, 1)
        Call(GetNpcPos, NPC_Lakitu_01, LVar0, LVar1, LVar2)
        ExecWait(N(EVS_GenericBombReaction))
#if !VERSION_JP
        Set(MV_ReactingNpc, -1)
    EndIf
#endif
```

I'm not sure

Reply:
For readability, I'd prefer fewer conditional blocks fwiw (keeping it as-is).